libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tab.component.scss

```diff
@@ -191,13 +247,21 @@
   display: flex;
   justify-content: space-between;
   flex-grow: 1;
+  align-items: var(--sky-override-vertical-tab-display-align-items, center);
```

Comment:
This allows us to get rid of some logic for positioning the right arrow for sectioned forms.